src/web_server.cpp

```diff
@@ -811,7 +811,7 @@ void handleOverrideGet(MongooseHttpServerRequest *request, MongooseHttpServerRes
     manual.getProperties(props);
     props.serialize(response);
   } else {
-    response->setCode(404);
+    response->setCode(200);
```

Comment:
Same as above, the `404` is correct

Reply:
I think the REST API should only answer 404 for not found element like /claims/{id}  /schedule/{id}
But not for getting claims/ or /override as it is still a real answer with needed info ( array is empty )